### DISCUSSION TITLE
If I link a user attachment in a gist, will it be deleted after 30 days like it wasn't hyperlinked?

### DISCUSSION THREAD

**Discussion by imthosea** [community/community#133472] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm currently using secret Gists to get my wikis peer-reviewed before I upload them publicly to Github Wikis. Currently, I'm uploading images into a github issue, then taking the link and putting it into the gist. However, I know that attachments are deleted after 30 days if they aren't linked somewhere. Even though the image embeds don't work in gists, does linking it there still stop the 30 day deletion?

**Comment by danialtavakoli** [comment, marked as answer]
No, linking to images in a gist does not prevent them from being deleted after 30 days. The 30-day limit applies to images uploaded directly to issues, regardless of whether they are linked elsewhere. To avoid deletion, consider hosting your images on a more permanent platform or directly within the repository if possible.
